pythonforandroid/recipes/python3/patches/py3.11.4_disable_parallel_build.patch

```diff
@@ -0,0 +1,15 @@
+--- Python-3.11.4/setup.py	2023-06-07 03:30:27.000000000 +0530
++++ Python-3.11.4.mod/setup.py	2023-07-09 15:04:57.209537129 +0530
+@@ -306,8 +306,10 @@
+         self.failed_on_import = []
+         self.missing = []
+         self.disabled_configure = []
+-        if '-j' in os.environ.get('MAKEFLAGS', ''):
+-            self.parallel = True
++
++        # to fix linking errors on python3.11
++        # if '-j' in os.environ.get('MAKEFLAGS', ''):
++        #    self.parallel = True
```

Comment:
If needed, instead of patching, we can properly set the `MAKEFLAGS` env var?

Reply:
Did you try to follow the advice?
Keeping our patches footprint smaller is a must for the maintainability of the recipes.